packages/jadmin/jadmin.js

```diff
@@ -241,3 +241,93 @@ yargs
   })
   .demandCommand(1)
   .parse(process.argv.slice(2))
+
+  yargs
+  .scriptName('jadmin')
+  .command({
```

Comment:
this `.command` call should be chained on after the previous one, rather than have a whole additional `yags...parse(...)` call. So like: 

```
yargs
  .scriptName('jadmin')
  .command({
    command: 'delete-user <userId>',
    ...
  })
  .command({
    command: 'delete-post <postId>',
    ...
  })
  .demandCommand(1)
  .parse(process.argv.slice(2))
```

Reply:
Awesome thanks! How's this?